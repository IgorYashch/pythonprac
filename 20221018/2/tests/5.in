:sum_of_four a b c d a+b+c+d
:mul_of_four x y z w x*y*z*w
sum_of_four 1 2 3 4
sum_of_four  5 5 5 5
mul_of_four  1 2 3 4
mul_of_four 9 8 7 4.0
quit "LaLaLa:{} BLAblaBla:{}"