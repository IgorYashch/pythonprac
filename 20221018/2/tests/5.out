10
20
24
2016.0
LaLaLa:3 BLAblaBla:7
